docs/examples/inception_casestudy/inception_casestudy.md

```diff
@@ -1271,3 +1271,84 @@ To prepare the data for training, follow these steps:
     
     export_model.predict(examples)
     ```
+
+## Troubleshooting
+
+**Q: What do I do if I get this error when trying to run PyTorch:**
+
+```psl
+hipErrorNoBinaryForGPU: Unable to find code object for all current devices!
+```
+
+Ans: The error denotes that the installation of PyTorch and/or other dependencies or libraries do not support the current GPU.
+
+**Workaround:**
+
+To implement a workaround, follow these steps:
+
+1. Confirm that the hardware supports the ROCm stack. Refer to the Hardware and Software Support document at [https://docs.amd.com](https://docs.amd.com).
+
+2. Determine the gfx target.
+
+    ```py
+    rocminfo | grep gfx
+    ```
+
+3. Check if PyTorch is compiled with the correct gfx target.
+
+    ```py
+    TORCHDIR=$( dirname $( python3 -c 'import torch; print(torch.__file__)' ) )
+    roc-obj-ls -v $TORCHDIR/lib/libtorch_hip.so # check for gfx target
+    ```
+
+:::{note}
+    Recompile PyTorch with the right gfx target if compiling from the source if the hardware is not supported. For wheels or Docker installation, contact ROCm support [6].
+:::
+
+**Q: Why am I unable to access Docker or GPU in user accounts?**
+
+Ans: Ensure that the user is added to docker, video, and render Linux groups as described in the ROCm Installation Guide at [https://docs.amd.com](https://docs.amd.com).
+
+**Q: Which consumer GPUs does ROCm support?**
+
+Ans: ROCm supports gfx1030, which is the Navi 21 series.
+
+**Q: Can I install PyTorch directly on bare metal?**
+
+Ans: Bare-metal installation of PyTorch is supported through wheels. Refer to Option 2: Install PyTorch Using Wheels Package in the section [Installing PyTorch](/ROCm/docs/how_to/pytorch_install/pytorch_install) of this guide for more information.
+
+**Q: How do I profile PyTorch workloads?**
+
+Ans: Use the PyTorch Profiler \[6\] to profile GPU kernels on ROCm.
+
+**Q: Can I run ROCm on Windows?**
+
+Ans: ROCm is not supported on Windows.
+
+## References
+
+C. Szegedy, V. Vanhoucke, S. Ioffe, J. Shlens and Z. Wojna, "Rethinking the Inception Architecture for Computer Vision," CoRR, p. abs/1512.00567, 2015

```

Comment:
A little bit more trouble, but references here are for the entire deep learning guide, it should be added the three guides we split it out to.

Reply:
Resolved and added:
+ porting 'Optimization' (missing Section) into Inception_casestudy.md
+ integrating missing image.08.png (into data/understand/deep_learning, used in Inception_casestudy.md)
+ Added brief AI/ML/Inferencing intro into ai_ml_inferencing.md (mostly a placeholder, until we add more examples)
+ Distributed references list to tensorflow, pytorch, and magma. References copied and pasted as heap, will require contextual trimming. 